synfig-studio/src/gui/resourcehelper.cpp

```diff
@@ -49,21 +49,29 @@ synfig::String studio::ResourceHelper::get_image_path()
 #endif
 
 	std::string imagepath;
-#ifdef _WIN32
-	imagepath=App::get_base_path()+'/'+IMAGE_DIR;
-#else
-	imagepath=IMAGE_DIR;
-#endif
 	char* synfig_root=getenv("SYNFIG_ROOT");
 	if(synfig_root) {
 		imagepath=synfig_root;
-		// Only class About didn't use the synfigstudio directory when using
+		//  Only class About didn't use the synfigstudio directory when using
 		//  SYNFIG_ROOT env variable. However, if it weren't set, it would use
 		//  IMAGE_DIR builtin variable that includes "synfigstudio". It means
 		//  that that About icon is in both folders. Therefore, it is safe to
 		//  choose this path.
 		imagepath+="/share/pixmaps/synfigstudio";
 	}
+	else {
+#ifdef CMAKE_BUILD
+        imagepath=App::get_base_path();
+        imagepath+="/share/pixmaps/synfigstudio";
+#else
+#   ifdef _WIN32
+        imagepath=App::get_base_path()+'/'+IMAGE_DIR;
```

Comment:
As far as we know what we do not broke Windows logic, you can just remove the whole Windows ifdef block (as it already undefines IMAGE_DIR):
```
#ifdef _WIN32
#	ifdef IMAGE_DIR
#		undef IMAGE_DIR
#		define IMAGE_DIR "share/pixmaps"
#	endif
#endif
```

and change this to:
```
imagepath=App::get_base_path()+"/share/pixmaps";
```